mne/io/base.py

```diff
@@ -657,7 +657,10 @@ def annotations(self):  # noqa: D401
     @property
     def filenames(self):
         """The filenames used."""
-        return tuple(self._filenames)
+        try:
+            return tuple(self._filenames)
+        except AttributeError:
+            return tuple([None])
```

Comment:
Don't change `base` change `eeglab.py` to set this to `[None]`, usually done in `super(RawEEGLAB, self).__init__(...)`, see e.g.:

https://github.com/mne-tools/mne-python/blob/master/mne/io/edf/edf.py#L169

Also this is a bit weird to set it to `None`, it should point to the underlying `.set` / `.mat` file so it's displayed properly e.g. in `raw.plot`.